### PR TITLE
[IMP] guidelines: complete list of directories and files

### PR DESCRIPTION
**Pull request by appstogrow**

### CODE REVIEW

File: website/Contribution/CONTRIBUTING.rst [hunk 1]
```diff
@@ -94,6 +94,7 @@ A module is organized in a few directories:
 * `examples/`: external files
   `lib/`, ...
 * `models/`: model definitions
+* `readme/`: e.g. CONTRIBUTORS.rst, DESCRIPTION.rst, DEVELOP.rst, HISTORY.rst, INSTALL.rst, ROADMAP.rst, USAGE.rst
```
Comment (luisg123v) [inline website/Contribution/CONTRIBUTING.rst:97]: This is giving examples, but it doesn't describe what the directory is for

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: website/Contribution/CONTRIBUTING.rst (hunk 1 above)
Comment (appstogrow) [reply website/Contribution/CONTRIBUTING.rst:97]: Would you prefer this?
* `readme/`: documentation

Comment (luisg123v) [reply website/Contribution/CONTRIBUTING.rst]: I would recommend you provide both a description and examples, e.g.:

files used to build the module's README.rst file, e.g. DESCRIPTION.rst, CONTRIBUTORS.rst. To see all supported files, check [the module template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Review by rvalyi — APPROVED** [review]

**Comment by appstogrow** [comment]
I have updated the PR so that section 1.4 and 1.7 have all the directories and main files as the [template module](https://github.com/OCA/maintainer-tools/tree/master/template/module), in the same order.

**Review by luisg123v — APPROVED** [review]
Excellent!

**Comment by appstogrow** [comment]
@rvalyi Do you approve the latest changes also?

**Review by flachica — CHANGES_REQUESTED** [review]
In * `README.rst`: comment:

is built from the files in the readme directory. You must not create or modify it

**Comment by norlinhenrik** [comment]
I have done as suggested by @flachica 
Can we merge?

**Comment by norlinhenrik** [comment]
I just changed 
pre_migration.py & post_migration.py
to
pre-migration.py & post-migration.py

**Comment by norlinhenrik** [comment]
I lookup CONTRIBUTING.rst quite regularly to make sure that I follow the OCA standards. And it is complicated to navigate to this PR all the time. Can we please merge?

**Comment by pedrobaeza** [comment]
@sbidoul @simahawk @eantones @dreispt can you please check?

**Comment by norlinhenrik** [comment]
If you would like to merge this PR, now is the time for it. I will delete all `appstogrow` repos.

BTW The last days I have installed ArchLinux, and I am impressed by the quality of their [wiki](https://wiki.archlinux.org/). For documentation, maybe OCA can have a wiki instead of git?

**Comment by sbidoul** [comment]
@norlinhenrik this looks good, thanks! Can you rebase to resolve the conflict?

**Comment by norlinhenrik** [comment]
I have rebased now.

**Pull request merged**